docs/mem_quick_start.ipynb

```diff
@@ -0,0 +1,597 @@
+    "* db flag is set : `google_ml_integration.enable_model_support` is set to on."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### To list all models available\n",
+    "This list includes the two pre built models:\n",
+    "* `textembedding-gecko`\n",
+    "* `textembedding-gecko@001`\n",
+    "\n",
+    "And any other model you may have created\n",
+    "\n",
+    "The alist_models function returns `List[AlloyDBModel]`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "results = await model_manager.alist_models()\n",
+    "print(results)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### To create a custom text embedding model\n",
+    "\n",
+    "To create a custom textembedding model you need to pass these parameters to the acreate_model function\n",
+    "\n",
+    "* model_id: A unique ID for the model endpoint that you define.\n",
+    "* model_provider: The provider of the model endpoint (`google` for vertexAI and `custom` for custom  hosted models).\n",
+    "* model_type: The model type (set this value to `text_embedding` for text embedding model endpoints or `generic` for all other model endpoints).\n",
+    "* model_qualified_name: The fully qualified name in case the model endpoint has multiple versions or if the model endpoint defines it.\n",
+    "\n",
+    "You can customize your model further with some optional parameters. For all the details and possibilities, check out the [reference doc](https://cloud.google.com/alloydb/docs/reference/model-endpoint-reference#google_mlcreate_model).\n",
+    "\n",
+    "\n",
+    "Important Note: The acreate_model function doesn't return any value directly.\n",
+    "You'll need to use alist_models to verify if your model was created successfully."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "await model_manager.acreate_model(\n",
+    "    model_id=\"textembedding-gecko@003\",\n",
+    "    model_provider=\"google\",\n",
+    "    model_qualified_name=\"textembedding-gecko@003\",\n",
+    "    model_type=\"text_embedding\",\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "##### You can also create a third-party custom text embedding model using these steps.\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Make sure the secret is added to the secretManager in your AlloyDB instance using psql / AlloyDB Studio."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "vscode": {
+     "languageId": "sql"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "CALL\n",
+    "  google_ml.create_sm_secret(\n",
+    "    'openai_key', '<secret>');"
```

Comment:
I'd maybe use the field names since it's hard to know what these fields refer to. We could remove this in favor of a link to the official docs
```suggestion
    "    'secret_id', 'secret_path');"
```

Reply:
I'd suggest we remove the exact specific way to do this.. but mention that in a Note / Disclaimer that users can call third party models, including from Hugging Face.

Added this for iteration later: [b/373303283](http://b/373303283)